mycroft/enclosure/gui.py

```diff
@@ -39,6 +39,15 @@ def __init__(self, skill):
         self.on_gui_changed_callback = None
         self.config = Configuration.get()
 
+    @property
+    def connected(self):
+        if self.skill.bus:
+            data = self.skill.bus.wait_for_response(
```

Comment:
I may have been awake for too long, but would this return a Message that has a data object rather than the data object directly?

Reply:
good catch, you are correct